addons/mail/static/src/js/extended_chat_window.js

```diff
@@ -44,6 +49,14 @@ return ChatWindow.extend({
     on_keydown: function (event) {
         event.stopPropagation();
     },
+    on_cilck_expand: function () {
+        event.preventDefault();
+        chat_manager.is_ready.then(this.discuss_expand.bind(this));
```

Comment:
why to you need to wait for chat_manager to be ready? The client action is already waiting for it.